plugins/woocommerce-admin/client/payments/payment-settings-banner.tsx

```diff
@@ -85,10 +90,32 @@ const WcPayBanner = () => {
 						size="12"
 						lineHeight="16px"
 					>
-						{ __(
-							'By using WooCommerce Payments you agree to be bound by our Terms of Service and acknowledge that you have read our Privacy Policy',
-							'woocommerce'
-						) }
+						{ interpolateComponents( {
+							mixedString: __(
+								'By using WooCommerce Payments you agree to be bound by our {{tosLink}}Terms of Service{{/tosLink}} and acknowledge that you have read our {{privacyLink}}Privacy Policy{{/privacyLink}} ',
+								'woocommerce'
+							),
+							components: {
+								tosLink: (
+									<Link
+										href="https://woocommerce.com/posts/terms-of-service-update/"
```

Comment:
I think the usual TOS link we use is `https://wordpress.com/tos/`

Reply:
Good catch! I'll fix it in another PR.

Payment changes are not related to this PR.  They're showing here because something is wrong after rebased. They have already merged into the trunk.

Reply:
A fix PR is ready: https://github.com/woocommerce/woocommerce/pull/33082